app/src/androidTest/java/com/example/sharingang/FakeCurrentUserProvider.kt

```diff
@@ -8,7 +8,7 @@ import kotlinx.coroutines.runBlocking
 import javax.inject.Inject
 
 class FakeCurrentUserProvider @Inject constructor(
-    private val userRepository: UserRepository
+    private val userRepository: UserRepository,
```

Comment:
```suggestion
    private val userRepository: UserRepository
```

Reply:
I'm not sure I understand this suggestion

Reply:
I think that there is an unnecessary comma (maybe I don't see it correctly on the review page)